js/classes/entities/midground/enemies/enemyBase.js

```diff
@@ -177,7 +178,7 @@ class EnemyBase {
             // if we're pursuing Chad, update the target position to Chad's position
             // also avoid changing direction in the middle of an attack animation
             if (this.enemy.state === "pursue" && this.enemy.action != "attacking") {
-                this.setTargetX(CHAD.pos.x);
+                this.setTargetX(CHAD.pos.x + CHAD.scaleBoundingBoxOffset().x);
```

Comment:
Does adding CHAD.scaleBoundingBoxOffset().x help find the center of Chad?

Reply:
`CHAD.pos.x + CHAD.scaleBoundingBoxOffset().x` is where Chad's bounding box starts, which is also where Chad's body starts. So, enemies will go towards x position of the left edge of Chad's body. It may be worth refining this position later, maybe by changing it to Chad's center like you say.